Lesson_29/connect_to_db.py

```diff
@@ -0,0 +1,50 @@
+
+import time
+import psycopg2
+
+def connect_db():
+    for _ in range(5):
+        try:
+            conn = psycopg2.connect(
+                dbname='my_test_db',
+                user='serhiiravliuk',
+                password='kabosina2517',
+                host='postgres',
+                port='5432'
+            )
+            return conn
+        except psycopg2.OperationalError:
+            time.sleep(2)
```

Comment:
You provided waiting in exception block. But it is not correct. In this case even after waiting will be done - exception will not be gone. You need to use timeout for connection for db.